.eslintrc.yaml

```diff
@@ -222,7 +222,7 @@ rules:
   react/no-will-update-set-state: error
   react/prefer-es6-class: off
   react/prop-types: off
-  react/react-in-jsx-scope: error
+  react/react-in-jsx-scope: off
```

Comment:
Maybe comment about why they're off, for future reference? We could also just remove them, not sure which one would be less confusing.

Reply:
A comment sounds good. I tend to prefer setting things to `off` just in case the defaults change.